lib/Backends/Interpreter/InterpreterNodes.cpp

```diff
@@ -2613,7 +2613,7 @@ void BoundInterpreterFunction::fwdMatMulInstFloatImpl(const MatMulInst *I) {
       // Perform DOT on the row an column.
       float sum = 0;
       for (dim_t i = 0; i < lhsDim[1]; i++) {
-        sum += float(lhs.at({x, i}) * rhs.at({i, y}));
+        sum += float(lhs.at({x, i})) * float(rhs.at({i, y}));
```

Comment:
Wohoooo!!!

Can we make the Accumulator configurable like in "
```
/// This is the quantized implementation of Convolution.
template <typename ElemTy, typename AccumulatorTy, typename BiasElemTy>
void BoundInterpreterFunction::fwdConvolutionInstQuantizedImpl("

This way we can control the Accumulation.

Also, a test case would be nice :)
```

Reply:
I wasn't planning to add any testing for this change.